assets/js/editor/words-counter/component.js

```diff
@@ -0,0 +1,45 @@
+import * as hooks from './hooks/ui/index.js';
+
+export default class Component extends $e.modules.ComponentBase {
+  getNamespace() {
+    return 'words-counter';
+  }
+
+  defaultHooks() {
+    return this.importHooks( hooks );
+  }
+
+  countWords() {
+    // Clone current document elements.
+    const html = elementor.documents.getCurrent().$element.clone();
+
+    // Remove the element edit tools elements.
```

Comment:
what is this? @matipojo

Reply:
The edit tools are included in the preview HTML and they disrupts the count.